scripts/check-deps.sh

```diff
@@ -28,8 +28,8 @@ for dep in $(LC_ALL=C sort -u "${tmpdir}/golist"); do
 		# ignore ourselves
 		github.com/influxdata/telegraf) continue;;
 
-		# dependency is replaced in go.mod
-		github.com/satori/go.uuid) continue;;
+    # dependency is replaced in go.mod
+    github.com/satori/go.uuid) continue;;
```

Comment:
can you restore the original spacing here? Thanks!

Reply:
Thanks for the hint. It's done.